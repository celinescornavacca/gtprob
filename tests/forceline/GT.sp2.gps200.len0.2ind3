(((((((((A87,(A107,A108)),(A174,A175)),((A165,(A184,A185)),(A51,A52))),(A0,((((A74,(A178,A179)),((A28,(A70,A71)),(A53,A54))),((((A91,(A172,A173)),A84),A78),A26)),(A4,A5)))),((((((A186,A187),(A196,A197)),(A194,A195)),A68),((A98,(A180,A181)),A13)),(((A2,(A44,(A79,(A122,A123)))),(((A9,(A72,A73)),(A8,(((A22,((A182,A183),(A153,A154))),(((A170,A171),A116),A63)),A10))),A1)),((((A20,(A85,A86)),((A103,A104),(A126,A127))),(((A109,(A190,A191)),(A161,A162)),((A30,A31),((A112,A113),(A93,A94))))),(A35,A36))))),((((((A95,A96),A83),((A89,(A168,A169)),(A120,A121))),(((A192,A193),A88),A46)),(A47,A48)),(A3,(((A61,A62),((A69,(A188,A189)),A43)),((A143,A144),A21))))),(((A27,(A105,A106)),(((A37,A38),A7),((A67,((A147,A148),A90)),(A81,A82)))),(((A132,A133),A77),((A6,(A155,A156)),(((((A18,A19),(A12,(A128,A129))),((((A118,A119),((A139,A140),A117)),A49),((((A134,A135),A131),((A145,A146),A130)),((A56,A57),A45)))),(A17,(A25,((A163,A164),A39)))),((((A80,(A124,A125)),(A114,A115)),(A64,(A101,A102))),(((A34,(A137,A138)),A33),(A99,A100)))))))),((((((A15,((A157,A158),A136)),((A24,(A151,A152)),A23)),((((((A75,A76),A41),(A59,A60)),(A97,(A159,A160))),A16),((A176,A177),(A55,((A166,A167),A58))))),(((A141,A142),A29),(A149,A150))),(((((A198,A199),A92),((A110,A111),A42)),A14),A11)),(A32,(A40,(A50,(A65,A66)))))),(((((((B4,B5),((((B177,B178),B54),B47),((B122,B123),B32))),(B10,((B24,((B136,B137),B53)),((B167,B168),B111)))),((B19,((B169,B170),B143)),(((((B190,B191),B144),(B118,(B138,B139))),B74),(B69,(B94,B95))))),(((B2,(((B192,B193),B30),((B145,B146),(B124,B125)))),(((B99,(B149,B150)),((B22,B23),((B20,B21),(((B11,B12),((((B57,((B152,(B155,B156)),(B82,(B89,B90)))),((B65,B66),(B46,((B132,B133),(B108,B109))))),(B180,B181)),((B25,(B198,B199)),((B42,B43),B31)))),((((B51,B52),(B87,B88)),(B28,(((B91,B92),(B120,B121)),(B44,B45)))),(B27,(B184,B185))))))),((B14,(((B116,B117),B35),((B77,B78),((B162,B163),B68)))),B9))),(B63,B64))),((B13,(((B126,B127),(B26,((B104,B105),B55))),(((B106,B107),(B79,B80)),(B102,B103)))),((B1,(((B17,B18),((B39,(B112,B113)),(B49,B50))),((B83,B84),B29))),(((B100,B101),B15),(B119,(B165,B166)))))),((((B147,(B173,B174)),(B75,B76)),((((((B130,B131),B48),(((B171,B172),B67),B36)),(B194,B195)),(B7,(B58,((B142,(B175,B176)),B73)))),(((B41,(B61,B62)),((B37,B38),B6)),((B186,B187),((B86,((B158,B159),B110)),B56))))),((B3,(((B93,(B153,B154)),(B70,((B157,(B164,(B196,B197))),(B134,B135)))),B8)),((((B160,B161),B96),((B114,B115),((B85,(B128,B129)),((B71,B72),B40)))),(((B179,(B188,B189)),(B140,B141)),(B0,((((B59,B60),B16),(B33,B34)),((B97,B98),(B81,(((B182,B183),B151),B148)))))))))));
