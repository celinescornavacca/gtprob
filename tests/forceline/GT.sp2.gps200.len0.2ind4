((((((((((A76,A77),(A60,(A192,A193))),((((A152,A153),(((A98,(A156,A157)),A91),A71)),(A89,A90)),(A33,(A186,A187)))),(((A69,(A109,A110)),A18),((((A81,A82),((((A107,A108),(A162,A163)),((A147,(A196,A197)),A133)),(A47,A48))),(((A113,A114),A103),((A72,A73),A24))),A2))),(((A16,(((A104,A105),A88),A52)),((((((A42,A43),(A143,A144)),((A37,(A62,A63)),((A131,A132),A28))),(((((A145,A146),((A118,A119),((A148,(A160,A161)),A117))),(A70,(A178,A179))),((A190,A191),A106)),(A136,A137))),((A84,A85),A68)),((A20,(A30,(A123,A124))),(A55,(A150,A151))))),((((A111,A112),(A35,(A53,(A56,(A93,A94))))),A29),A26))),(((((((A101,A102),(A54,(A74,A75))),A8),((A64,((A95,A96),A66)),((A198,A199),A170))),(((A40,(A67,(A172,A173))),((A12,A13),(A4,A5))),(A19,(A25,((A176,A177),(A86,A87)))))),((((A41,((A182,A183),A122)),(A57,(((A141,A142),((A166,A167),A140)),(A168,A169)))),A1),((((A78,A79),((A158,A159),A149)),((((A45,A46),((A99,A100),A65)),(((A194,A195),A171),A92)),(((A138,A139),A80),(A22,A23)))),((A180,A181),A44)))),(A3,(A6,A7)))),(((A9,(A17,(A174,A175))),((((A58,A59),A51),(A31,A32)),((A164,A165),A83))),(A10,A11))),(((((A14,(((A34,(A188,A189)),A27),A15)),((A120,A121),A36)),((A97,(A134,A135)),(A49,A50))),(A21,(((A129,A130),A61),(A127,A128)))),A0)),(((A184,A185),(A115,A116)),(((A154,A155),(A125,A126)),(A38,A39)))),(((((B65,(B119,B120)),((((((B162,B163),(B19,(B62,(B174,B175)))),(B26,B27)),((B30,((B186,B187),B123)),B28)),B5),B4)),(((((((B91,B92),(B41,B42)),(((B144,B145),B59),((B70,B71),(B85,((B94,B95),B93))))),B9),(B2,(B29,((B172,B173),(B60,B61))))),((((B151,B152),((((B103,B104),(B87,B88)),(B39,B40)),(B20,(B44,(B75,B76))))),((B83,B84),(((B21,(B105,(B188,B189))),((B130,(B170,B171)),(((B166,B167),B106),B57))),(((B126,B127),((B17,((B178,B179),((B190,B191),((B137,B138),(B79,(B128,B129)))))),(B81,(B89,B90)))),(((B101,B102),B22),(B108,(B159,B160))))))),(B15,B16))),(((B12,(((B164,B165),(B111,B112)),(B99,B100))),(((((B143,(B148,B149)),B66),B11),((((B184,B185),B114),(B63,(B121,B122))),(((B117,B118),B80),(B33,B34)))),B7)),(B0,(B3,((B10,(B194,B195)),(B51,B52))))))),((B1,((B109,B110),B107)),(((((((B77,B78),((B139,B140),B56)),(((B192,B193),B23),B18)),((B64,(B182,B183)),(((B198,B199),(B141,B142)),B67))),((B176,B177),((B68,B69),B58))),B6),((((B157,B158),B136),(B53,B54)),(((B115,B116),B82),(B153,B154)))))),(((((B47,B48),B35),((B124,B125),B55)),((B45,B46),((((B196,B197),B161),((B131,B132),B43)),((B155,B156),B72)))),(((B73,B74),B36),((B8,((((B49,B50),((B97,B98),B96)),((B135,(B146,B147)),B31)),(B113,((B168,B169),B150)))),((((B180,B181),B13),(((B32,((B133,B134),B86)),(B37,B38)),B14)),(B24,B25)))))));
