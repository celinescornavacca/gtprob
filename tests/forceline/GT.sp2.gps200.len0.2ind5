((((((A21,A22),((((((A186,A187),(A113,A114)),(A31,A32)),((A47,A48),(A40,(A87,A88)))),((A196,A197),A122)),A5)),((((A57,(A178,A179)),(((((A121,(A182,A183)),A13),A7),((A8,((A14,A15),(((A60,A61),((A172,A173),A162)),(A34,A35)))),(((A165,(A168,(A169,A170))),A33),(A105,(A119,A120))))),(A25,((A148,(A149,A150)),(A77,A78))))),(((A2,(((A96,A97),((A50,A51),(A30,((A157,A158),A39)))),((((A192,A193),A29),((A95,(A133,A134)),A65)),((A71,(A84,A85)),((A127,A128),A58))))),(((((A171,(A184,A185)),A145),A140),(A89,A90)),(A188,A189))),((A135,(A139,(A174,A175))),(A38,(A110,A111))))),(((A190,A191),A70),A62))),((((A43,((A82,A83),((A101,A102),(A72,A73)))),((A115,A116),((A93,A94),A59))),((((A146,A147),A49),(((A166,A167),A161),(A53,A54))),(A17,A18))),((A0,(((A44,A45),((A103,A104),(A74,A75))),(((A108,A109),(A79,A80)),(A76,((A136,A137),A112))))),((((((A194,A195),A67),A52),(((A125,A126),(A143,A144)),(A55,(A153,A154)))),(((A1,(A3,((A23,A24),(A81,(A151,(A159,A160)))))),(A63,A64)),(A9,A10))),((A36,A37),A28))))),((((((A46,(A66,((A152,(A163,A164)),A86))),(A16,((A129,A130),A56))),(A4,((A117,A118),(A106,A107)))),((A98,(A123,A124)),(A19,A20))),((((A198,A199),((A99,A100),(A91,A92))),(A26,A27)),(((((A180,A181),A138),(A155,A156)),((A141,A142),(A176,A177))),(A41,A42)))),((A131,A132),((A6,(A11,A12)),(A68,A69))))),(((B15,(B53,B54)),(((B181,B182),B107),((B112,B113),B42))),(((((((B35,(B140,B141)),(B38,(B39,B40))),((((B83,((B147,B148),B105)),B16),(B49,B50)),B14)),((B28,B29),(B24,(B45,((B108,B109),(((B88,(B118,B119)),B74),((B168,B169),B135))))))),(((B106,(B122,(B183,(B194,B195)))),((B87,(B116,B117)),((B123,(B164,B165)),B86))),((((B171,B172),(B84,B85)),((((((B47,B48),(B30,B31)),B19),B4),((B177,B178),(B59,B60))),(((B32,((B75,B76),B52)),(B65,B66)),(B2,(((B132,(B161,B162)),B68),B10))))),(B0,(((B143,B144),((B159,B160),B131)),B67))))),(((B1,(((B57,B58),(B23,(B101,(B102,B103)))),((B166,B167),(B44,(B69,B70))))),((((((B93,(B104,(B179,B180))),B11),(B13,(((B71,B72),(B114,B115)),(B73,((B198,B199),B128))))),(((B91,B92),(B18,((((B133,B134),(B151,B152)),B77),B27))),((B41,(B153,B154)),((B120,B121),((B149,B150),B97))))),((B8,(B138,B139)),(B5,(B95,B96)))),((((B61,B62),B34),B20),(((B126,B127),((B79,B80),(B98,(B186,B187)))),(((B9,(B43,(B51,(B190,B191)))),(((B196,B197),((B110,B111),(B99,B100))),(B142,(B188,B189)))),((B157,B158),(B163,(B170,(B192,B193))))))))),((B55,B56),(((B173,B174),((((B136,B137),(B155,B156)),B33),((B25,B26),(B81,B82)))),((B21,B22),(((B184,B185),B78),B64)))))),(((B36,B37),(((B46,(B175,B176)),B6),(B17,(((B129,B130),B94),B63)))),(((B12,(B89,B90)),B3),((B145,B146),(B7,(B124,B125))))))));
