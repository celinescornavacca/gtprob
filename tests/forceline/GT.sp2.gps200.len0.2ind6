(((((A0,(((((A107,(A194,A195)),(A174,A175)),((((((A113,A114),A86),A5),(A63,A64)),((((A48,(A78,(A83,(A108,(A129,A130))))),A43),((A23,(A166,A167)),A19)),((A141,A142),A25))),(A2,(((A190,A191),(A105,A106)),((A39,A40),A28))))),((((A79,A80),A41),((A122,A123),A47)),A9)),(((A131,A132),A46),(((A8,(((A180,A181),A27),A14)),((((A90,A91),A50),((((A188,A189),((A158,A159),A45)),A20),A16)),((A34,A35),(((A65,A66),A56),A37)))),((A95,A96),((A198,A199),(A67,(A143,A144)))))))),((A29,A30),A1)),(((A51,A52),(A17,((A170,A171),A42))),(((A97,((((A154,A155),A137),A112),A109)),(A117,A118)),A15))),((((A156,A157),(A32,(A184,A185))),(((((A60,A61),(A73,A74)),((A81,(A82,(A196,A197))),A24)),((A44,((((A115,A116),A62),A59),A49)),(A6,(A21,A22)))),(((A10,A11),((A12,(((A172,A173),A146),((A121,(A133,A134)),A89))),(A126,(A153,(A192,A193))))),A3))),(((((((((A99,A100),A94),(A53,((A92,(A135,A136)),(A162,A163)))),A26),A4),((((A178,A179),A18),(A70,A71)),((((A101,A102),(A145,(A168,A169))),A31),((A72,(A160,A161)),(A36,(A68,A69)))))),((((A110,A111),((A119,A120),A98)),(((A186,A187),((A33,((A76,A77),(A103,A104))),(A87,A88))),A13)),(A7,((A75,((A138,A139),((A124,A125),(A164,A165)))),((A54,A55),(A57,A58)))))),((A84,A85),((((A140,(A149,A150)),(A182,A183)),(A176,A177)),((A151,A152),(A127,A128))))),(A38,(A93,(A147,A148)))))),(((((B103,B104),((B23,B24),((B176,B177),(B147,B148)))),(((B180,B181),(((((B116,(B198,B199)),B75),((B58,B59),(B182,B183))),(B70,((B157,B158),B122))),(B26,((B63,B64),B50)))),(B154,B155))),((((((((B54,(B164,B165)),(B170,B171)),((B118,B119),(B172,B173))),((B98,(B166,B167)),B57)),(((B100,B101),(B72,((B184,B185),(B143,B144)))),B18)),(B4,B5)),(((B9,((B36,(B125,B126)),(B51,(B68,B69)))),((B133,B134),B120)),((B16,B17),B10))),(((((((B192,B193),B34),B22),(((B112,B113),((B131,B132),B92)),(B78,B79))),(((B66,B67),B46),((B71,(B84,B85)),(B45,(B159,(B194,B195)))))),((((B19,(B135,B136)),(B29,B30)),(B44,(B123,B124))),(B42,((B186,B187),B88)))),((B32,(B38,(B61,((B102,(B149,(B152,B153))),B80)))),B8)))),((((((B178,B179),((B48,B49),B14)),B1),((B2,B3),(((B114,B115),((B145,B146),(B43,(B93,B94)))),(((B55,(B95,B96)),(((B196,B197),(B90,B91)),B56)),B7)))),(((((B128,(B188,B189)),B97),B39),B20),(B0,((((B139,B140),B127),((B12,B13),B6)),(((B129,B130),(B86,B87)),(B52,B53)))))),(((B15,((B21,((B73,B74),B25)),(B60,((B160,B161),B77)))),((B168,B169),B121)),((((((((B47,(B105,(B190,B191))),B37),(B110,B111)),B33),(((B117,(B174,B175)),(B108,B109)),B76)),((B35,(((B99,((B162,B163),B156)),(B150,B151)),(B83,(B137,B138)))),(B31,(B81,B82)))),((B62,((B141,B142),B89)),(B27,B28))),(B11,((B40,B41),(B65,(B106,B107)))))))));
