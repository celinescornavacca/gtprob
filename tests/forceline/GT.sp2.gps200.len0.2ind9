(((((A3,((((A168,A169),A34),A32),(A147,A148))),(((((A63,(A131,A132)),(A100,A101)),((A71,(A86,A87)),A25)),A2),((((A127,A128),A33),(A29,(A54,A55))),(((A42,A43),(A115,(A171,A172))),(((A192,A193),A135),(A121,(A141,A142))))))),((((A5,((((A157,A158),A92),(A136,A137)),A12)),(((A51,(A53,(A69,(A73,A74)))),(((A183,(A190,A191)),(A198,A199)),A48)),(A17,(A35,A36)))),(((((A96,A97),A76),A62),(A186,A187)),(A113,A114))),(((A9,A10),((((A122,(A151,A152)),A61),(((A15,(((A64,(A88,((A105,A106),A91))),(A145,A146)),(A129,A130))),(A13,(A75,(A107,A108)))),(A14,(A16,(A26,(A153,A154)))))),((((A11,(A21,(A143,A144))),((A111,A112),((A94,(A123,A124)),((A89,A90),(A44,(A68,(A196,A197))))))),((A23,A24),((A161,A162),A70))),((A18,((A77,(A149,A150)),(A103,A104))),(A4,(A72,(A83,A84))))))),((((A19,A20),((A177,A178),(A120,(A166,A167)))),(A40,((A109,A110),(A60,(((A188,A189),A170),(A118,(A119,(A179,A180)))))))),(((A80,A81),A41),(((A173,A174),A52),(((A175,A176),(A78,A79)),A37))))))),(((A0,(A7,A8)),(((((A102,(A163,(A184,A185))),A82),(A116,A117)),(A85,(A98,A99))),(((A1,(A6,(A47,((A95,(A155,A156)),A65)))),(A38,(A58,A59))),(((A27,A28),(((A49,A50),(((A194,A195),(A139,A140)),(A66,A67))),A22)),(((A181,A182),(A93,(A159,A160))),(A138,(A164,A165))))))),((A125,A126),(((A56,A57),((A39,(A133,A134)),A30)),((A45,A46),A31))))),((((((((((B57,B58),((B26,(B147,B148)),B20)),(((B169,B170),((B32,(B42,B43)),(((B134,B135),((B91,B92),B79)),(B36,((B52,(((B190,B191),B166),B119)),B46))))),(B11,(((B194,B195),B86),(B61,((B69,B70),(B95,B96))))))),(((B24,B25),B16),((B157,B158),B63))),(((B161,(B167,B168)),B33),((B99,B100),(B71,B72)))),(((B4,((B28,((B188,B189),B29)),((B59,B60),(B87,B88)))),((((B164,B165),(B81,B82)),B27),((B53,B54),((((B174,B175),B103),(B141,B142)),((B192,B193),(B78,((B155,B156),B101))))))),((B23,(B40,B41)),(B132,B133)))),(B8,(((B137,B138),(B74,B75)),((B89,B90),(B97,B98))))),((B7,((((((B153,B154),B136),B129),B66),B62),(B51,((B182,B183),(B111,(B116,B117)))))),(((((B122,B123),B104),B102),B34),(B47,B48)))),B0),(((((B31,(((B186,B187),(B162,B163)),(B124,B125))),((((B120,B121),B19),(B107,B108)),((B30,((B93,B94),B49)),B2))),B1),((B151,B152),(B21,B22))),(((((B198,B199),B35),(B5,((B130,B131),(B37,(B159,B160))))),((B84,B85),B12)),((B3,((((B176,B177),B76),B15),(((((((B139,B140),(B113,B114)),B64),(B143,B144)),B10),((B17,B18),((B55,B56),((B50,(B109,B110)),((B172,B173),B77))))),(((((B38,(B145,B146)),((B105,B106),((B127,B128),B65))),(B13,(((B44,B45),((B112,(B126,(B196,B197))),B80)),B14))),((((((B184,B185),B171),B118),B73),B39),B9)),(B178,B179))))),((((B83,(B115,(B149,B150))),(B180,B181)),(B67,B68)),B6))))));
