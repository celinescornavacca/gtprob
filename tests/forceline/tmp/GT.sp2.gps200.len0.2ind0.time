36.65user 0.58system 0:37.46elapsed 99%CPU (0avgtext+0avgdata 256160maxresident)k
16inputs+8outputs (0major+191810minor)pagefaults 0swaps
