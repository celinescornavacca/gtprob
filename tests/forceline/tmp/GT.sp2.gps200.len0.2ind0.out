-1939.9519219
